pkg/registry/apis/ofrep/register.go

```diff
@@ -168,29 +278,25 @@ func writeResponse(statusCode int, result any, logger log.Logger, w http.Respons
 	}
 }
 
-func (b *APIBuilder) stackIdFromEvalCtx(body []byte) string {
+func (b *APIBuilder) stackIdFromEvalCtx(body []byte) int64 {
 	// Extract stackID from request body without consuming it
 	var evalCtx struct {
 		Context struct {
-			StackID int32 `json:"stackId"`
+			StackID int64 `json:"stackId"` // TODO -- replace with namespace "stackId" ONLY makes sense in cloud
 		} `json:"context"`
 	}
 
 	if err := json.Unmarshal(body, &evalCtx); err != nil {
 		b.logger.Debug("Failed to unmarshal evaluation context", "error", err, "body", string(body))
-		return ""
+		return 0
 	}
 
 	if evalCtx.Context.StackID <= 0 {
 		b.logger.Debug("Invalid or missing stackId in evaluation context", "stackId", evalCtx.Context.StackID)
-		return ""
+		return 0
 	}
 
-	return strconv.Itoa(int(evalCtx.Context.StackID))
```

Comment:
replaced with the standard namespace parser; but likely should avoid direct reference to stacks in any case